.meteor/packages

```diff
@@ -17,23 +17,15 @@ es5-shim@4.8.0
 
 # Collections
 aldeed:collection2
-wekan-cfs-standard-packages
+cfs:standard-packages
 cottz:publish-relations
 dburles:collection-helpers
 idmontie:migrations
-matb33:collection-hooks
 matteodem:easy-search
-mongo@1.11.0
+mongo@1.14.0
 mquandalle:collection-mutations
 
 # Account system
-kenton:accounts-sandstorm
```

Comment:
But but... removing Sandstorm accounts support???

Reply:
Not sure. This was removed by @xet7 in the commit `40265144` made on Fri Jun 25 14:11:53 2021 +0300.